.github/ISSUE_TEMPLATE.md

```diff
@@ -1,24 +1,28 @@
-Thanks for submitting an issue! 
+Thanks for submitting an issue! Please fill in as much of the template below as
+you can.
 
--------------BUG REPORT --------------------
-1. Fill in as much of the template below as you can. 
+------------- BUG REPORT TEMPLATE --------------------
 
-2. What kops version are you running? use `kops version`
+1. What kops version are you running? Use `kops version`?
 
-3. What Kubernetes version are you running? use `kubectl version`
+1. What Kubernetes version are you running? Use `kubectl version`?
 
-4. What cloud provider are you using?
+1. What cloud provider are you using?
 
-5. What commands did you execute (Please provide cluster manifest `kops get --name my.example.com`, if available) and what happened after commands executed?
+1. What commands did you execute, and what happened after the commands executed?
 
-6. What you expected to happen:
+1. Can you provide cluster manifest `kops get --name my.example.com -oyaml`?
 
-7. How can we to reproduce it (as minimally and precisely as possible):
+1. What you expected to happen?
 
-8. Anything else do we need to know:
+1. How can we to reproduce it (as minimally and precisely as possible)?
 
--------------FEATURE REQUEST --------------------
+1. Can you run the `kops` command with the `-v 10` flag and provide logs?
```

Comment:
Please run the command with most verbose logging by adding the `-v 10` flag and paste the logs into this report.